packages/python-packages/protocol-stub-generator/tests/test_serializer.py

```diff
@@ -0,0 +1,75 @@
+import os
+import sys
+import unittest
+import yaml
+from protocol.protocol_models import ProtocolClientView, ProtocolOperationView
+from protocol.parse_yml import create_python_name
+
+
+sys.path.append(os.path.join(os.path.dirname(os.path.abspath(__file__)), ".."))
+
+
+PATH = os.getenv("TESTJSONPATH")
+
+
+class TestCases(unittest.TestCase):
+    def test_client_name(self):
+        with open(PATH) as f:
+            data = yaml.safe_load(f)
+        create_python_name(data)
+        client = ProtocolClientView.from_yaml(data)
+        assert client.Name == "Batch Document Translation Client"
```

Comment:
it looks like we asserting against Document Translation in these tests. In that case we should indicate somewhere in a comment on this test that you must use that service in particular to run these tests

Reply:
Will do!